docs/source/torch.rst

```diff
@@ -488,5 +488,6 @@ Utilities
     compiled_with_cxx11_abi
     result_type
     can_cast
-
     promote_types
+    set_deterministic
```

Comment:
Sorry for not being very familiar with the history here, but is there already something that will generate the doc for set_deterministic and is_deterministic once they're added to the rst?

Reply:
Maybe, but I'm not aware if so.